API.md

```diff
@@ -0,0 +1,67 @@
+# Communications Template Service Api
+
+This page documents the Communications Template service API endpoints
+
+## Classification Types
+The following endpoints are associated with the classification types that are used to specify which template to use when communicating.
+
+### Create a classification type
+
+* `POST /classificationtype/GEOGRAPHY`
+
+### Delete a classification type
+
+* `DELETE /classificationtype/GEOGRAPHY`
+
+### GET a classification type
+
+* `GET /classificationtype/GEOGRAPHY` will return the classification type or None if it doesn"t exist.
+
+### GET all classification types
+
+* `GET /classificationtype` will return a list of all classification types
+
+## Communications Templates
+
+## Create a template
+
+* `POST /template/cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87` with a valid template object in the request json.
+
+## Delete a template
+
+* `DELETE /template/cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87`
+
+## Update a template
+
+* `PUT /template/cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87` with a valid template object in the request json.
+
+## Get templates by classifiers
+
+* `GET /template` will return a list of communications templates which match the classifiers, these are passed in the request json.
+
+Example Response
+```json
+[{"classification": {"GEOGRAPHY": "NI"}, 
```

Comment:
This would probably look a bit cleaner if you indented it in a more json-like way. i.e. 
```
[{
  "classification": {
    "GEOGRAPHY": "NI"
  }, 
  "id": "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef89",
  "label": "test data",
  ...
}]
```